src/sentry/demo/data/errors/python/concat_str_none.json

```diff
@@ -1,377 +1,377 @@
       ]
     },
-    "extra": {
-      "inventory": {"hammer": 1, "nails": 1, "wrench": 1},
-      "sys.argv": ["/usr/local/bin/flask", "run", "--host=0.0.0.0", "-p", "8080"]
-    },
-    "fingerprint": ["{{ default }}"],
-    "grouping_config": {
-      "enhancements": "eJybzDhxY3J-bm5-npWRgaGlroGxrpHxBABcTQcY",
-      "id": "newstyle:2019-10-29"
-    },
-    "hashes": ["05ddd70c1c4fe6ce654dff4ed8f8bfbc"],
-    "key_id": "1512188",
-    "level": "error",
-    "location": "app.py",
-    "logger": "",
-    "metadata": {
-      "filename": "app.py",
-      "function": "checkout",
-      "type": "TypeError",
-      "value": "cannot concatenate 'str' and 'NoneType' objects"
-    },
-    "modules": {
-      "blinker": "1.4",
-      "certifi": "2020.12.5",
-      "click": "7.1.2",
-      "flask": "1.0.2",
-      "flask-cors": "3.0.7",
-      "gunicorn": "0.14.2",
-      "itsdangerous": "1.1.0",
-      "jinja2": "2.11.3",
-      "markupsafe": "1.1.1",
-      "numpy": "1.16.6",
-      "pg8000": "1.12.5",
-      "pip": "20.0.2",
-      "psycopg2-binary": "2.8.4",
-      "python": "2.7.18",
-      "python-dotenv": "0.12.0",
-      "pytz": "2020.4",
-      "sentry-sdk": "0.19.2",
-      "setuptools": "44.1.0",
-      "six": "1.15.0",
-      "sqlalchemy": "1.3.15",
-      "typing": "3.7.4.3",
-      "urllib3": "1.26.3",
-      "werkzeug": "1.0.1",
-      "wheel": "0.34.2",
-      "wsgiref": "0.1.2"
-    },
-    "received": 1616715154.288638,
-    "request": {
-      "url": "https://app.example.com/checkout",
-      "method": "POST",
-      "data": {
-        "cart": [
-          {"id": "wrench", "name": "Wrench", "price": 500},
-          {"id": "wrench", "name": "Wrench", "price": 500}
-        ]
-      },
-      "headers": [
-        ["Accept", "*/*"],
-        ["Accept-Encoding", "gzip, deflate"],
-        ["Content-Length", "139"],
-        ["Content-Type", "application/json"],
-        ["Forwarded", "for=\"34.122.208.80\";proto=https"],
-        ["Host", "aj-flask-m3uuizd7iq-uc.a.run.app"],
-        ["User-Agent", "python-requests/2.25.1"],
-        ["X-Cloud-Trace-Context", "cc4b36f5ce29a7324c14547568fa219e/957919981805984821"],
-        ["X-Forwarded-For", ""],
-        ["X-Forwarded-Proto", "https"]
-      ],
-      "env": {"SERVER_NAME": "0.0.0.0", "SERVER_PORT": "8080"},
-      "inferred_content_type": "application/json"
-    },
-    "sdk": {
-      "name": "sentry.python",
-      "version": "0.19.2",
-      "integrations": [
-        "argv",
-        "atexit",
-        "dedupe",
-        "excepthook",
-        "flask",
-        "logging",
-        "modules",
-        "sqlalchemy",
-        "stdlib",
-        "threading"
-      ],
-      "packages": [{"name": "pypi:sentry-sdk", "version": "0.19.2"}]
-    },
-    "timestamp": 1616715154.279259,
-    "title": "TypeError: cannot concatenate 'str' and 'NoneType' objects",
-    "transaction": "checkout",
-    "type": "error",
-    "version": "7"
-  }
+    "headers": [
```

Comment:
@roggenkemper why is event JSON changing?

Reply:
I have no idea. It is exactly the same as before. might have something to do with indentations? but nothing has changed